bin/garden-init

```diff
@@ -189,6 +189,8 @@ if [ -z "$nonDebian" ]; then
 
 	# Copy Garden Linux repository key
 	cp "$keyringPlain" "$targetDir/etc/apt/trusted.gpg.d/gardenlinux.asc"
```

Comment:
You might want to use the usual `install` tool to do that.

```suggestion
	install -o root -g root -m 644 "$keyringPlain" "$targetDir/etc/apt/trusted.gpg.d/gardenlinux.asc"
```

Or install the file via the existing `file.include` mechanism,

Reply:
Good point, we should use `install` instead. However, using the feature system could be too late within the overall process. We should keep it here.